module/zfs/vdev_label.c

```diff
@@ -1061,6 +1095,14 @@ vdev_uberblock_load_done(zio_t *zio)
 
 	if (zio->io_error == 0 && uberblock_verify(ub) == 0) {
 		mutex_enter(&rio->io_lock);
+
+		dprintf("vdev_uberblock_load_done best_txg %llu txg %llu "
+		    "best_timestamp %llu timestamp %llu\n",
+		    (u_longlong_t)cbp->ubl_ubbest->ub_txg,
+		    (u_longlong_t)ub->ub_txg,
+		    (u_longlong_t)cbp->ubl_ubbest->ub_timestamp,
+		    (u_longlong_t)ub->ub_timestamp);
```

Comment:
Let's keep this in the final version, I've definitely wanted to see this in the dprintf logs in the past.